docs/xdr/features/collect/integrations/endpoint/sekoiaio.md

```diff
@@ -22,36 +44,82 @@ A link to download the latest version of the agent is available in the descripti
 
 ### Installation
 
-The Endpoint Detection Agent is a Windows binary which you can easily install on Windows systems, after having created a dedicated intake on SEKOIA.IO XDR. The following command must be executed **as an administrator**:
+The Endpoint Detection Agent is a binary which you can easily install on Windows or Linux systems, after having created a dedicated intake on SEKOIA.IO XDR.
 
-```shell
-agent.exe -install -intake-key <INTAKE_KEY>
-```
+=== "Windows"
 
-To make sure the agent is successfully installed as a service you can run the following command:
+    The following command must be executed **as an administrator**:
 
-```shell
- Get-Service SEKOIAEndpointAgent
-```
+    ```shell
+    agent.exe -install -instake_key <INTAKE_KEY>
+    ```
+
+    To make sure the agent has been successfully installed as a service you can run the following command:
+
+    ```shell
+    Get-Service SEKOIAEndpointAgent
+    ```
+
+=== "Linux"
+
+    If `auditd` is running on the machine we must disable it before installing the linux agent:
 
-Once installed, the agent collects Windows event logs from ETW (Event Tracing for Windows), normalizes them and sends them to SEKOIA.IO. The protocol used to send events is HTTPS (443).
+    ```shell
+    sudo systemctl stop auditd
+    sudo systemctl disable auditd
+    ```
+
+    Now that `auditd` is disabled we can install the agent:
+
+    ```shell
+    chmod +x ./agent
+    sudo ./agent -install -instake_key <INTAKE_KEY>
+    ```
+
+    To make sure the agent has been successfully installed as a service you can run the following command:
+
+    ```shell
+    sudo systemctl status SEKOIAEndpointAgent.service
+    ```
+
+ Once installed, the agent collects event logs, normalizes them and sends them to SEKOIA.IO. The protocol used to send events is HTTPS (443).
```

Comment:
Adding the domain/ip could be usefull to add/debug firewall issues

Reply:
The domain is `sekoia.io` so easy to guess. Regarding the IP I'm afraid it will change and we will forget to update the documentation accordingly.

Reply:
isn't it https://intake.sekoia.io with a static IP ?

Reply:
I added it